CHANGELOG.md

```diff
@@ -1,5 +1,70 @@
 # Node.js Agent for Google Cloud Trace ChangeLog
 
+## 2017-02-28, Version 1.0.0-rc1, @matthewloring
+
+### Notable changes
+
+This version introduces a new API for creating custom trace spans as well as an API for writing custom plugins
```

Comment:
Please state clearly that this is a semver major change – can we add a list of behaviour change?. Most users – who weren't using the custom tracing API – would be minimally affected.